### DISCUSSION TITLE
【求助】看到了绝区零表情包，可以告诉我怎么获取吗

### DISCUSSION THREAD

**Discussion by poolofwater** [share121/inter-knot#2226] [其他]
如题<img src="https://img-static.mihoyo.com/communityweb/upload/6b88559cbba96a1fa965e5412736370d.png" alt="兔躯一震" width="75" >

**Comment by Hejin1212** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >emmm

**Comment by ADAD33** [comment]
翻一下，找到一个叫Dave-12138的大佬的帖，里面可以下载<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Kamisato100** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/c537b8edbffea525b48f438b084271f1_7376084967646241027.png" alt="11号-看着你" width="75" >

**Comment by Maixiaofanqie** [comment]
往下翻一翻应该找得到大佬发的教程，就是一个油猴插件<img src="https://bbs-static.miyoushe.com/static/2023/04/28/6475fffe75e1042eb1c68ac615adfdaf_2494846049179701109.png" alt="米仔_得意" width="75" >

**Comment by LEARoTY** [comment]
这位大佬的帖子，有一个插件下载就好啦https://share121.github.io/inter-knot/?code=21d88ca1f481b2febb52&article=2049<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by guozichang2007** [comment]
不知道欸<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by HeisenbergMrWhite** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >1

**Comment by sj49193646** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/27fb9141bd6d9a72a5ebff8b6ae10a00.png" alt="黑塔 你说咩啊" width="75" >嘿嘿嘿

**Comment by Qweulin** [comment]
哈哈哈<img src="https://img-static.mihoyo.com/communityweb/upload/76cc94063d9c0efb2335e8487299cd0a.png" alt="银狼 吹泡泡" width="75" >

**Comment by Mr123517** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >好唉

**Comment by Mr123517** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >好唉

**Comment by Mr123517** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >好唉

**Comment by ASTOU2311** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Klein124284420** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/1b7c688ec343e27aa83a7061d82e46a2.png" alt="闭嘴" width="75" >

**Comment by Klein124284420** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/baaaf420abb173dac46a466b5115e3ce.gif" alt="米游姬 感谢" width="75" >6

**Comment by chen3537** [comment]
这
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/cfc12c6552ad36ddf88bd651e90da441_8544373733813917056.png" alt="丽都漫步-苍角" width="75" >

**Comment by wuyueji** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >( •̀ ω •́ )y
